e2e/tests/functional/plugins/displayLayout/displayLayout.e2e.spec.js

```diff
@@ -0,0 +1,107 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+const { test, expect } = require('../../../../pluginFixtures');
+const { createDomainObjectWithDefaults, setStartOffset, setFixedTimeMode, setRealTimeMode } = require('../../../../appActions');
+
+test.describe('Testing Display Layout @unstable', () => {
+    let sineWaveObject;
+    test.beforeEach(async ({ page, openmctConfig }) => {
+        await page.goto('./', { waitUntil: 'networkidle' });
+        await setRealTimeMode(page);
+        sineWaveObject = await createDomainObjectWithDefaults(page, {
+            type: 'Sine Wave Generator',
+            name: "Test Sine Wave Generator"
+        });
+
+        const { myItemsFolderName } = openmctConfig;
+        await page.locator(`text=${myItemsFolderName}`).first().click();
+    });
+    test('alpha-numeric widget telemetry value exactly matches latest telemetry value received in real time', async ({ page }) => {
+        await createDomainObjectWithDefaults(page, {
+            type: 'Display Layout',
+            name: "Test Display Layout"
+        });
+        await page.locator('[title="Edit"]').click();
+
+        await page.locator('.c-tree__item__view-control.c-disclosure-triangle').click();
```

Comment:
I think we should leave some small comments here documenting the steps so it's less confusing to folks who may not be as familiar.

`// Enter edit mode`
`// Expand the 'My Items' folder in the tree`

etc.

This will also make it easier to preserve the original intent of the test when we will inevitably have to update them to work cross-environment

Reply:
yup, this makes sense! I added comments breaking down the flow of the test